### ISSUE TITLE
The extension seems to not be working with phpstan/extension-installer

### ISSUE THREAD

**Issue by andreasciamanna** [szepeviktor/phpstan-wordpress#26]
Here is my **global** `composer.json`:

```json
{
  "config": {
      "sort-packages": true
  },
  "require": {
  },
  "require-dev": {
    "giacocorsiglia/stubs-generator": "^0.5.0",
    "phpstan/extension-installer": "^1.0",
    "phpstan/phpstan": "^0.12.50",
    "szepeviktor/phpstan-wordpress": "^0.6.6"
  }
}
````

(I'm not using `giacocorsiglia/stubs-generator` at the moment, but it's installed in case I need to generate some stubs)

My inspection fails on a file where line &numero;2 is `if(is_admin() || defined('XMLRPC_REQUEST')):`.

The error is `Function is_admin not found.`.

In fact, this and many other files produce similar errors multiple times for basic functions such as `is_admin`, `__`, `add_action`, etc.

Any lead on what am I missing?

**Comment by szepeviktor** [comment]
Thank you for the report!
Please share your `phpstan.neon` file. I think something is missing/in wrong order.

**Comment by szepeviktor** [comment]
And please remember that you have to pay the price of static analysis: write testable code :)

**Comment by andreasciamanna** [comment]
Hi, thanks for the quick reply!

> And please remember that you have to pay the price of static analysis: write testable code

Static code analysis had little, if anything to do with "testable code".
PHPStan behaves like a compiler. My code does "compile".
The problem is with the dependencies.
WordPress is one of them, and your package, for which I thank you, is meant to solve the issue. 

Maybe you may want to consider the option where I might be "bit more" an experienced programmer ;)

I'm dealing with very old legacy code here which, among the many issues, may have some testability problems, but again, that's not supposed to be a problem for a tool such as PHPStan (unless you use a very loose definition of "testable").

I do agree, however, that there might be some configuration issues from my side.

In fact, I'm pretty sure that's the case, and that's why I posted a request here.

I'm new to PHPStan and I'm still figuring some of its settings and how it works.

Anyway, here is my neon file:

```yaml
parameters:
  level: 0
  bootstrapFiles:
    - tests/phpstan/phpstan-bootstrap.php
  paths:
    - classes
    - compatibility
    - inc
    - lib
    - menu
    - modules
  excludes_analyse:
    - lib/twig
```

`tests/phpstan/phpstan-bootstrap.php` has nothing in it yet.

It's there just in case I need to add some definition.

**Comment by szepeviktor** [comment]
Okay!

Just add water :)

```yaml
includes:
    - vendor/szepeviktor/phpstan-wordpress/extension.neon
```

[At the top.](https://github.com/szepeviktor/phpstan-wordpress/blob/master/example/phpstan.neon.dist#L8)

**Comment by szepeviktor** [comment]
@andreasciamanna Actually I would gladly help you to improve your legacy code base. Please send me [an email](mailto:viktor@szepe.net).

**Comment by andreasciamanna** [comment]
Great. That worked.

I thought `phpstan/extension-installer` would take care of including this file.

Since it doesn't, I've removed it as it's useless at this point.

Thanks, Viktor!

**Comment by szepeviktor** [comment]
@andreasciamanna That is very stange!

Could you remove the above config and add `error_log('PHPSTAN-EXTENSION:'.$package->getName());` before
vendor/phpstan/extension-installer/blob/master/src/Plugin.php:112 ?

**Comment by andreasciamanna** [comment]
@szepeviktor there is not such `extension-installer` folder.

It seems I'm getting a Phar version of the package:

```
vendor/phpstan
└── phpstan
    ├── LICENSE
    ├── README.md
    ├── bootstrap.php
    ├── composer.json
    ├── conf
    │   └── bleedingEdge.neon
    ├── phpstan
    ├── phpstan.phar
    └── phpstan.phar.asc
```

**Comment by szepeviktor** [comment]
> there is not such `extension-installer` folder.

Then you are not using that package :(

```
composer require --dev phpstan/extension-installer
```

**Comment by andreasciamanna** [comment]
Right. I forgot I just removed it. 🤦

I've made the change you asked, but I didn't know where to expect the output.

So I ran `vendor/bin/phpstan analyse`, and now the errors are gone (?).

BTW, running `composer install` shows `PHPSTAN-EXTENSION:szepeviktor/phpstan-wordpress`.

I guess I may have messed up something with Composer when I reported this issue.

Sorry for the waste of time @szepeviktor  :(

And thank you, of course!
